packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -459,6 +442,114 @@ export class TablePageObject<T extends TableRecord> {
         }
     }
 
+    /**
+     * @param columnIndex The index of the column to the left of a divider being dragged. Thus, this
+     * can not be given a value representing the last visible column index.
+     * @param deltas The series of mouse movements in the x-direction while sizing a column.
+     */
+    public dragSizeColumnByRightDivider(
+        columnIndex: number,
+        deltas: number[]
+    ): void {
+        const divider = this.getColumnRightDivider(columnIndex);
+        if (divider === null) {
+            throw new Error(
+                'The provided column index has no right divider associated with it.'
+            );
+        }
+        divider.setAttribute('active', 'true');
```

Comment:
`active` is a CSS class now rather than an attribute, right? Is this line still needed? If so, should there be an equivalent line setting `active` back to `false` at the end of the function?

Same question for `dragSizeColumnByLeftDivider` below.

Reply:
Removed.